File: src/main/java/de/arrobait/antlers/editor/AntlersQuoteHandler.java
package de.arrobait.antlers.editor;

import com.intellij.codeInsight.editorActions.SimpleTokenSetQuoteHandler;
import de.arrobait.antlers.psi.AntlersTokenSets;

public class AntlersQuoteHandler extends SimpleTokenSetQuoteHandler {
    public AntlersQuoteHandler() {
        super(AntlersTokenSets.STRINGS);
    }
}

File: src/main/java/de/arrobait/antlers/editor/AntlersQuoteHandler.kt
package de.arrobait.antlers.editor

import com.intellij.codeInsight.editorActions.SimpleTokenSetQuoteHandler
import de.arrobait.antlers.psi.AntlersTokenSets

class AntlersQuoteHandler : SimpleTokenSetQuoteHandler(AntlersTokenSets.STRINGS)
